File: exo2/exo2.py
#exercices
#by marvin
#beWeb

#On commence par constituer le tableau et les autres variables 
tableau = [3,20,15,28,6,52,9]
switch = ""
sorted = False
print("base : " + str(tableau))
while sorted == False :
    moved = False
    i = 0
#Si l index courant est inferieur a la longueur du tableau si cette valeur et plus grande on switch
    while i < len(tableau) -1:
        if tableau[i+1] > tableau[i]:
            switch = tableau[i] 
            tableau[i] = tableau[i + 1] 
            tableau[i + 1] = switch
            moved = True
        i += 1
    print("boucle : " + str(tableau))
    if moved == False:
        sorted = True
print(tableau)

File: exo2/exo2.php
<?php
 $tableau =(3,25,1,6,23,56);
 $permut = 0;
 $i =0;
 while {
     $sorted = true;
     for ($i = 0; $i < count($tableau)-1; $i++) {
         if ($tableau[$i] > $tableau[$i + 1]) {
             $permut = $tableau[$i];
             $tableau[$i] = $tableau[$i + 1];
             $tableau[$i + 1] = $permut;
             $sorted = false;
		print_r ($tableau);
         }
     }
 }
 print_r ($tableau);
?>
